src/insert/comments-expanded-advert.ts

```diff
@@ -53,6 +89,9 @@ const isEnoughSpaceForAd = (rightColumnNode: HTMLElement): boolean => {
 	return rightColumnNode.offsetHeight >= minHeightToPlaceAd;
 };
 
+const isEnoughCommentsForAd = (commentsColumn: HTMLElement): boolean =>
+	commentsColumn.childElementCount > 5;
```

Comment:
If we insert an advert after every fourth ad and it's acceptable for an advert to be at the bottom of the comments, should this be `commentsColumn.childElementCount >= 4;`?

Reply:
We've checked this with Alex and have decided that an advert should never be at the bottom of the comments. I've introduced a check to make sure that this doesn't happen.